src/index.ts

```diff
@@ -124,3 +126,45 @@ async function _replacePlaceholder(wdioConf: string) {
     fileString = fileString.replace(/%baseUrl%/g, BASE_URL)
     await fs.writeFile(wdioConf, fileString)
 }
+
+function _getUsedPackageManager(): PackageManager {
+    const alternativeManagers = [PackageManager.pnpm, PackageManager.pnpm]
```

Comment:
this should be `const alternativeManagers = [PackageManager.pnpm, PackageManager.yarn]`, right? :)

Reply:
Funny that it still worked when I tried `yarn create`. Maybe they are more compatible than `pnpm` and `npm`